fe/src/main/java/org/apache/doris/clone/DynamicPartitionScheduler.java

```diff
@@ -275,19 +280,25 @@ private void executeDynamicPartition() {
                 // scheduler time should be record even no partition added
                 createOrUpdateRuntimeInfo(olapTable.getName(), LAST_SCHEDULER_TIME, TimeUtils.getCurrentFormatTime());
                 RangePartitionInfo rangePartitionInfo = (RangePartitionInfo) olapTable.getPartitionInfo();
+                if (rangePartitionInfo.getPartitionColumns().size() != 1) {
+                    // currently only support partition with single column.
+                    iterator.remove();
+                    continue;
+                }
+
```

Comment:
The single column check work is done when create table, no need to check here

Reply:
It is just a self-defence~